.buildkite/ftr_configs.yml

```diff
@@ -56,6 +56,7 @@ disabled:
   - x-pack/test/plugin_api_perf/config.js
   - x-pack/test/screenshot_creation/config.ts
 
+defaultQueue: 'n2-4-spot-2'
```

Comment:
Can you actually change this to `n2-4-spot`? We'll phase out `n2-4-spot-2`. It was originally introduced for the last big spot instance roll out so that it would be easier to move a subset of the spot instances back to non-spot if the rollout went badly. It's just a duplicate of the other one.

Reply:
Ah, cool